Vapi 1094 forking api (#625)
Bandwidth/api-docs: site/docs/voice/webhooks/mediaStreamStarted.mdx

---
id: mediaStreamStarted
title: Media Stream Started
slug: /voice/webhooks/mediaStreamStarted
description: A general overview of Bandwidth's Media Stream Started Webhook
keywords:
  - bandwidth
  - voice
  - webhook
  - streaming
  - start
hide_title: false
image: ../../static/img/bandwidth-logo.png
---

> This event is sent to the url specified when sending a [`<StartStream>`][1] verb.

Reviewer: Here too
```suggestion
This event may be sent to the url specified when sending a [`<StartStream>`][1] verb.
```

| callId                  | The call id associated with the event. |
| callUrl                 | The URL of the call associated with the event. |
| direction               | The direction of the call. Either `inbound` or `outbound`. The direction of a call never changes. |
| enqueuedTime            | (optional) If [call queueing](/apis/voice#operation/createCall) is enabled and this is an outbound call, time the call was queued, in ISO 8601 format. |
| eventTime               | The approximate UTC date and time when the event was generated by the Bandwidth server, in ISO 8601 format. This may not be exactly the time of event execution. |
| eventType               | The event type, value is `mediaStreamStarted` |
| from                    | The provided identifier of the caller: can be a phone number in E.164 format (e.g. +15555555555) or one of `Private`, `Restricted`, `Unavailable`, or `Anonymous`. |
| mediaStream             | Details about the stream that was started |
| mediaStream.destination | The destination URL to which the stream is sending media |
| mediaStream.id          | The unique id of the stream |

> | mediaStream.name        | The name of this stream. If the `name` attribute was specified in the [`StartStream`][1] verb, then this will be the value of that attribute, otherwise it will be the value auto generated by the system. |

Reviewer 2: ```suggestion
| mediaStream.name        | The name of this stream. If the `name` attribute was specified in the [`StartStream`][1] verb, then this will be the value of that attribute, otherwise it will default to the stream id.
```
